src/Compilers/Helpers/SimpleAnalyzerAssemblyLoader.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Reflection;
+
+namespace Microsoft.CodeAnalysis
+{
+    /// <summary>
+    /// Loads analyzer assemblies from their original locations in the file system.
+    /// Assemblies will only be loaded from the locations specified when the loader
```

Comment:
And things are even more complicated in the CoreCLR world, where hosts have more control over native image loading.